Author: Andreas Tille <tille@debian.org>
        Aaron M. Ucko <ucko@debian.org>
Last-Update: 2023-01-15 20:56:15 -0500
Description: Find ncbi vdb at Debian location

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,8 +54,8 @@ set(CMAKE_MODULE_PATH
     "${CMAKE_CURRENT_SOURCE_DIR}/build"
 )
 
-set( VDB_BINDIR "${CMAKE_BINARY_DIR}/../ncbi-vdb" CACHE PATH "Location of NCBI VDB build directory" )
-set( VDB_INCDIR "${CMAKE_SOURCE_DIR}/../ncbi-vdb/interfaces" CACHE PATH "Location of NCBI VDB include directory" )
+set( VDB_BINDIR "/usr/include/ncbi-vdb" CACHE PATH "Location of NCBI VDB build directory" )
+set( VDB_INCDIR "/usr/include/ncbi-vdb/interfaces" CACHE PATH "Location of NCBI VDB include directory" )
 
 include(CTest)
 enable_testing()
--- a/tools/loaders/bam-loader/CMakeLists.txt
+++ b/tools/loaders/bam-loader/CMakeLists.txt
@@ -27,7 +27,7 @@ if( NOT WIN32)
 
 	add_compile_definitions( __mod__="tools/bam-loader" )
 
-	include_directories( ${CMAKE_SOURCE_DIR}/../ncbi-vdb/interfaces/ext/ ./) # zlib.h
+	include_directories( /usr/include/ncbi-vdb/interfaces/ext/ ./) # zlib.h
 
 	#set(CMAKE_BUILD_TYPE RelWithDebInfo)
 	if (APPLE)
--- a/tools2/shared/cmake/Findncbi-vdb.cmake
+++ b/tools2/shared/cmake/Findncbi-vdb.cmake
@@ -44,7 +44,8 @@ elseif (NOT ncbi-vdb_FIND_QUIETLY)
 endif ()
 
 find_path (ncbi-vdb_INCLUDE_DIR "vdb/manager.h"
-    PATHS   "../ncbi-vdb/interfaces"
+    PATHS   "/usr/include/ncbi-vdb/interfaces"
+            "../ncbi-vdb/interfaces"
             "../../ncbi-vdb/interfaces"
             "../../../ncbi-vdb/interfaces"
             "../../../../ncbi-vdb/interfaces"
